pepper-apis/dsm-core/src/main/resources/liquibase/juniper-ourheart/PEPPER-1057-our_heart_setup.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="pegah" id="new-juniper-study-instance-role">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM instance_role WHERE name='juniper_study';
+            </sqlCheck>
+        </preConditions>
+        <insert tableName="instance_role">
+            <column name="name" value="juniper_study"/>
+        </insert>
+    </changeSet>
+    <changeSet id="our-health-ddp-instance-insert" author="pegah">
+        <insert tableName="ddp_instance">
+            <column name="instance_name" value="juniper-ourheart"/>
+            <column name="study_guid" value="juniper-ourheart"/>
+            <column name="display_name" value="Our Health"/>
+            <column name="is_active" value="1"/>
+            <column name="bsp_group" value="Endocrine/MPG"/>
+            <column name="bsp_collection" value="SC-57013"/>
+            <column name="bsp_organism" value="1"/>
+            <column name="collaborator_id_prefix" value="PN"/>
+            <column name="auth0_token" value="0"/>
+            <column name="billing_reference" value="CO 8201079"/>
+        </insert>
+    </changeSet>
+    <changeSet id="our-health-ddp-group-insert" author="pegah">
+        <insert tableName="ddp_group">
+            <column name="name" value="juniper-ourheart"/>
+            <column name="description" value="Juniper Our Health"/>
```

Comment:
If I understood your comment above, this should probably be `juniper-ourhealth`, since Our Health is the study group?

Reply:
I understand that from a study / group point of view, but I was not sure if everyone in the ourhealth group that get access to Juniper would also want access to the second study. In my POV groups are for user-access setups mainly  and I was thinking lets keep the our health's general name out of this, so the name shows people in this group will have access to our heart. 
This group - portal - study separation that is in Juniper and is not in Pepper is a bit confusing, I don't mind changing the name of these fields either way

Reply:
Perhaps I am just confused, but based on what you said above I was thinking that there was an Our Health study group, and Our Heart was a study in that group.
But it seems like each Juniper study will have it's own study group (which is primarily a way to share access control across studies administered by the same DSM users)?
If so, then calling this study two different things seems confusing.

Reply:
I think I confused you. Sorry!
I take what I said back because I just realized our health folks will not have access to DSM, it will just be GP, so we can name it what we want since we are not going to decide who is a member of our heart vs our health